### PR TITLE
added validation to the VDB name #59

### PR DESCRIPTION
**Pull request by lhein**
Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

### CODE REVIEW

**Comment by lhein** [comment]
good points

**Comment by lhein** [comment]
I fixed the mentioned issues except the tests which will be handled as part of #45

File: src/utils.ts [hunk 1]
```diff
@@ -84,4 +86,20 @@ export function replaceTemplateName(dsConfig: IDataSourceConfig, dsName: string,
...
 	return dsConfigNew;
-}
\ No newline at end of file
+}
+
+export function validateName(name: string): string {
+	if (/^[a-z0-9]{4,253}$/.test(name)) {
+		return undefined;
+	} else {
+		return "The entered name does not comply with the naming conventions. [a-z0-9]";
```
Comment (apupier) [inline src/utils.ts]: I think we need to mention that it needs to have a length between 4 and 253 too

Comment (lhein) [reply src/utils.ts]: right

**Comment by lhein** [comment]
yes, planned for friday and next week.

**Pull request merged**